src/Aquarius.ONE.ClientSDK/Common/Schedule/ScheduleApi.cs

```diff
@@ -12,15 +11,17 @@ namespace ONE.Common.Schedule
 {
     public class ScheduleApi
     {
-        public ScheduleApi(PlatformEnvironment environment, bool continueOnCapturedContext, RestHelper restHelper)
+        public ScheduleApi(PlatformEnvironment environment, bool continueOnCapturedContext, RestHelper restHelper, bool throwAPIErrors = false)
         {
             _environment = environment;
             _continueOnCapturedContext = continueOnCapturedContext;
             _restHelper = restHelper;
+            _throwAPIErrors = throwAPIErrors;
         }
         private PlatformEnvironment _environment;
         private bool _continueOnCapturedContext;

```

Comment:
It looks like a couple methods were missed - like Get Activitie

Reply:
GetOneActivity() method is done